rpm/podman.spec

```diff
@@ -24,6 +24,8 @@
 
 %if %{defined fedora}
 %define build_with_btrfs 1
+# QEMU isn't packageed for CentOS Stream / RHEL
```

Comment:
shouldn't this say qemu-system is not packaged?
because it seem it now depend on qemu-kvm for Centos/RHEL?

Reply:
done.